client/my-sites/themes/atomic-transfer-dialog.tsx

```diff
@@ -225,8 +225,8 @@ class AtomicTransferDialog extends Component< AtomicTransferDialogProps > {
 }
 
 export default connect(
-	( state: IAppState ) => {
-		const siteId = getSelectedSiteId( state );
+	( state: IAppState, ownProps?: AtomicTransferDialogProps | { siteId: number } ) => {
+		const siteId = getSelectedSiteId( state ) ?? ownProps?.siteId;
```

Comment:
Question: Sometimes `siteId` will be passed and some not? If that's the case why the passed property doesn't have precedence against the selector value?

Reply:
Good point, but this change is not required anymore; it is a leftover from previous commits, so I will revert the change.

Reply:
Commit that reverts the changes https://github.com/Automattic/wp-calypso/pull/81412/commits/8c0df9725b35f14a54ee8131267c7d8ef0e1cba2